src/index.js

```diff
@@ -111,10 +111,15 @@ export const createOffline = (userConfig: $Shape<Config> = {}) => {
     // create store
     const store = createStore(reducer, preloadedState, enhancer);
 
-    const baseReplaceReducer = store.replaceReducer.bind(store);
-    store.replaceReducer = function replaceReducer(nextReducer) {
-      return baseReplaceReducer(enhanceReducer(nextReducer, config));
-    };
+    if (config.enhanceReplaceReducer !== false) {
```

Comment:
this make HMR opt-in feature, when it always was enabled by default. Could we make it opt-out instead?
This meants we need to have the opposite: `disabledEnhanceReplaceReducer`

Reply:
This is explicitly checking for `false` in order to make it opt-out. If unset the behaviour will remain as it was so this shouldn't be a breaking change.

I can still invert the logic and change the name if you'd prefer?

Reply:
oh I get it! Yes please invert the logic. We have 3 possibilities now: true, false, undefined. Where true and undefined do the same thing 😅 . I prefer having false and undefined doing the same thing.

Reply:
No problem :smile: It's just a matter of personal preference, so I'm happy to go with whatever convention you like to follow.

I'll make the change as soon as I find some time.